src/core_scan/Core_scan.ml

```diff
@@ -968,6 +968,15 @@ let scan_with_exn_handler (config : Core_scan_config.t) :
     let timed_rules =
       Common.with_time (fun () -> rules_from_rule_source config)
     in
+    (* Configure HTTP mocking based on network file provided by environment
+       variable. An environment variable is used to (1) avoid needing to thread
+       a command line flag throughout and (2) for ease of testing, given there
+       isn't a reason to use this option in a production environment *)
+    Sys.getenv_opt "SEMGREP_MOCK_NETWORK"
```

Comment:
I am not a huge fan of the magic string constant but I looked at our Python code and it seems pretty standard so I guess it's ok

Reply:
luv me magic constants